resources.go

```diff
@@ -96,13 +99,19 @@ func Provider() tfbridge.ProviderInfo {
 			"bigip_ltm_snatpool":                    {Tok: f5BigIPResource(f5BigIPLTMMod, "SnatPool")},
 			"bigip_ltm_virtual_address":             {Tok: f5BigIPResource(f5BigIPLTMMod, "VirtualAddress")},
 			"bigip_ltm_virtual_server":              {Tok: f5BigIPResource(f5BigIPLTMMod, "VirtualServer")},
+			"bigip_ltm_profile_client_ssl":          {Tok: f5BigIPResource(f5BigIPLTMMod, "ProfileClientSsl")},
+			"bigip_ltm_profile_server_ssl":          {Tok: f5BigIPResource(f5BigIPLTMMod, "ProfileServerSsl")},
 			"bigip_sys_dns":                         {Tok: f5BigIPResource(f5BigIPSysMod, "Dns")},
 			"bigip_sys_iapp":                        {Tok: f5BigIPResource(f5BigIPSysMod, "IApp")},
 			"bigip_sys_ntp":                         {Tok: f5BigIPResource(f5BigIPSysMod, "Ntp")},
 			"bigip_sys_provision":                   {Tok: f5BigIPResource(f5BigIPSysMod, "Provision")},
 			"bigip_sys_snmp":                        {Tok: f5BigIPResource(f5BigIPSysMod, "Snmp")},
 			"bigip_sys_snmp_traps":                  {Tok: f5BigIPResource(f5BigIPSysMod, "SnmpTraps")},
 			"bigip_sys_bigiplicense":                {Tok: f5BigIPResource(f5BigIPSysMod, "BigIpLicense")},
+			"bigip_ssl_certificate":                 {Tok: f5BigIPResource(f5BigIPSslMod, "Certificate")},
+			"bigip_ssl_key":                         {Tok: f5BigIPResource(f5BigIPSslMod, "Key")},
+			"bigip_as3":                             {Tok: f5BigIPResource(f5BigIPMainMod, "As3")},
+			"bigip_do":                              {Tok: f5BigIPResource(f5BigIPMainMod, "Do")},
```

Comment:
Out of curiosity, what is "do"? If it's an acronym, the capitalized name would be "DO".